kalite/distributed/static/js/distributed/user/views.js

```diff
@@ -362,7 +362,7 @@ window.ToggleNavbarView = BaseView.extend ({
     initialize: function() {
 
         _.bindAll(this);
-        this.listenTo(this.model, "change", this.render);
+        this.listenTo(this.model, "change:client_server_time_diff", this.render);
```

Comment:
I typed in this.statusModel.changedAttributes() into the inspect element console to find out if there were any attributes that changed on page load.
client_server_time_diff was the only one that had a change so I just tried that.

Would it be better to create a new attribute or something on statusModel?